### ISSUE TITLE
Add new object options

### ISSUE THREAD

**Issue by IonicaBizau** [jillix/jQuery-json-editor#7]

**Comment by IonicaBizau** [comment]
We should be able to add new fields in objects, not only edit them (#6). 

```js
roles: {
    label: "Access",
    edit: true, // this is already working
    addField: true
}
```

**Comment by silviubogan** [comment]
For reference, while working on this issue I discovered [a new jsfiddle issue](https://code.google.com/p/chromium/issues/detail?id=543485) related to a very recent change in Chrome behavior.

**Comment by IonicaBizau** [comment]
Fixed via #23. :tada:
